modules/snooker/src/main/ts/ephox/snooker/api/Sizes.ts

```diff
@@ -62,6 +63,18 @@ const redistribute = function (table: Element, optWidth: Option<string>, optHeig
 
 };
 
+const isPercentSizing = Sizes.isPercentSizing;
```

Comment:
Wasn't sure if these maybe should be in `TableSize`... I left them here though, as `TableSize` won't tree shake exports and this still seemed like a relevant place.